docs/reference/commandline/cli.md

```diff
@@ -96,28 +96,36 @@ variables.
 ### Configuration files
 
 By default, the Docker command line stores its configuration files in a
-directory called `.docker` within your `$HOME` directory. However, you can
-specify a different location via the `DOCKER_CONFIG` environment variable
-or the `--config` command line option. If both are specified, then the
-`--config` option overrides the `DOCKER_CONFIG` environment variable.
-For example:
-
-    docker --config ~/testconfigs/ ps
-
-Instructs Docker to use the configuration files in your `~/testconfigs/`
-directory when running the `ps` command.
+directory called `.docker` within your `$HOME` directory.
 
 Docker manages most of the files in the configuration directory
 and you should not modify them. However, you *can modify* the
 `config.json` file to control certain aspects of how the `docker`
 command behaves.
 
-Currently, you can modify the `docker` command behavior using environment
+You can modify the `docker` command behavior using environment
 variables or command-line options. You can also use options within
 `config.json` to modify some of the same behavior. When using these
-mechanisms, you must keep in mind the order of precedence among them. Command
-line options override environment variables and environment variables override
-properties you specify in a `config.json` file.
+mechanisms, you **must** keep in mind the order of precedence among them.
+Command line options override environment variables and environment variables
+override properties you specify in a `config.json` file.
+
+
+#### Change the `.docker` directory
+
+To specify a different directory, use the `DOCKER_CONFIG`
+environment variable or the `--config` command line option. If both are
+specified, then the `--config` option overrides the `DOCKER_CONFIG` environment
+variable.
+
+For example:
+
+    docker --config ~/testconfigs/ ps
+
+Instructs Docker to use the configuration files in your `~/testconfigs/`
+directory when running the `ps` command.
+
```

Comment:
Perhaps in this location, we should explain that the `--config` flag is only applying to the command that's run (i.e., it's not updating the default configuration, and only applies to the command that you ran with that flag set).

After explaining that, I think we should add a _second_ example that's using the `DOCKER_CONFIG` environment variable; we can show that example using `export DOCKER_CONFIG=........`, and below the example add a note that if they want to persist that configuration, they can set it in, e.g. their `~/.profile` (or `~/.bashrc`, or whatever flavour of shell they're using)

Reply:
I took a shot at the command, how does it look @thaJeztah ?